packaged_releases/HISTORY.md

```diff
@@ -1,6 +1,59 @@
 Release History
 ===============
 
+2.0.8 (2017-06-13)
```

Comment:
@troydai, mind adding an entry for the greatest 2.0.9 azure-cli-core? Or, i can submit a PR after this get merged